python/dgl/nn/pytorch/gt/spatial_encoder.py

```diff
@@ -140,107 +142,63 @@ def __init__(self, num_kernels, num_heads=1, max_node_type=1):
         self.gaussian_stds = nn.Embedding(1, num_kernels)
```

Comment:
Seems that it suffices to use `nn.Parameter` here?

Reply:
yes, you're right